### DISCUSSION TITLE
Kingart dimmer / Tasmota

### DISCUSSION THREAD

**Discussion by stoffel53** [arendst/Tasmota#13510] [Q&A] [answered]
I bought a glass brandless dimmer (Alieexpress), Flashed Tasmota on it, setup the Wifi and configured it (PS-16-DZ (58)). So far so good. But... ït doesn't work. Manually (on the glass) it does work, but not on the tasmota screen. Do I miss something?

**Comment by sfromis** [comment, marked as answer]
Well, your dimmer may not be a PS-16-DZ, which is a very particular device, not a generic thing with a "standard" build. To make Tasmota work on your thing, you should find out how it is to be configured.

**Reply by stoffel53** [reply]
I followed the instructions as given in the link below:

https://tasmota.github.io/docs/devices/KingArts-touch-Dimmer/

**Reply by sfromis** [reply]
You wrote "brandless dimmer" which means that anything could be inside, and may have been changed since that rather old page was created. The configuration for https://templates.blakadder.com/PS-16-DZ.html is very narrowly focused on having analyzed a particular device years ago. Watching the console with `Weblog 3` while trying things may provide clues about what happens.

**Reply by stoffel53** [reply]
Thx very much, I will have it a try.

**Comment by stoffel53** [comment]
Finally I am able to make some time to try. First I have to say that I am a newbee. I am trying to set up een Home Assistant configuration in combination with a number of Sonoff switches flashed with tasmota. Also a hikvision camera (ONVIF protocol), my TADO set (central heating setup) and my solar panels. This is working now. I learned a lot during the process. Now the dimmer...
As I said, the flashing with Tasmota 10 worked well. The webgui looks fine and as far I can see in the console the us of the Webgui works well. For instance when I toggle, the console answers correct. There is no reaction at all in the console of actions with the hardware (the actual dimmer).
The hardware works partly: The led does not work. The switch (the touchbutton in the middle) works well, I can toggle the switch ON / OFF. The left touchbutton should go down in my opinion, it actually goes up. The right touchbutton does the opposite, it goes down.

I don't know if I do thing right with WebLog 3. I typed it in the console and than tried the buttons: no respons at all.

**Reply by sfromis** [reply]
A good start would be to check how the hardware and control input/outputs is built, if there is another MCU or how it is connected, if it is at all similar to PS-16-DZ. Tasmota can't do anything with the hardware unless you configure it exactly right.

**Comment by stoffel53** [comment]
Yes it looks like a second MCU, because Tasmota is up and running (and it's recognised by HA), but not the hardware. Separatly the hardware works on his own (more or less). I think I let it go and place it there were it is not necessary for automation. Thanks anyway for your help, it is greatly appreciated.
